azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/StorageAccountEncryptionKeySource.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.storage;
+
+import com.microsoft.azure.management.apigeneration.LangDefinition;
+
+/**
+ * Azure storage account encryption key sources.
+ */
+@LangDefinition
+public class StorageAccountEncryptionKeySource {
+    /** Static value Blob for StorageAccountEncryptionKeySource. */
+    public static final StorageAccountEncryptionKeySource MICROSOFT_STORAGE = new StorageAccountEncryptionKeySource("Microsoft.Storage");
```

Comment:
since this is a hand-written expandable enum with a single string mapping, why not derive from [`ExpandableStringEnum`](https://github.com/Azure/azure-sdk-for-java/blob/master/azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/arm/ExpandableStringEnum.java)?